pages.pt-BR/common/7za.md

```diff
@@ -1,20 +1,20 @@
 # 7za
 
 > Um compactador de arquivos com alta taxa de compressão.
-> Versão do `7z` com suporte para poucos tipos de arquivamento/compressão.
+> Versão com menos opções de tipos de arquivamento/compressão em relação ao `7z`.
```

Comment:
As I mentioned [here](https://github.com/tldr-pages/tldr/pull/2913#discussion_r275704259), I think it would be clearer to say something like

> Versão compacta do 7z, com suporte para menos tipos de arquivamento/compressão.

Also, to avoid splitting discussions across multiple conversations, please hold creating PRs until an existing discussion reaches consensus, otherwise the thread cannot be properly followed, making the rationale for the change less clear.

Reply:
Hi @waldyrious 

Sure, my bad :man_facepalming: